EasyFinance.Domain/Financial/DefaultCategory.cs

```diff
@@ -0,0 +1,51 @@
+﻿using EasyFinance.Domain.Models.Financial;
+using System.Collections.Generic;
+
+namespace EasyFinance.Domain.Financial
+{
+    public class DefaultCategory
```

Comment:
I don't think that is necessary created a new class for this, you can use the existent category and add predefined instances as you made here.

Reply:
Done